Services/Administration/OJS.Services.Administration.Data/Implementations/RoleDataService.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OJS.Services.Administration.Data.Implementations;
+
+using OJS.Data;
+using OJS.Data.Models.Users;
+using OJS.Services.Common.Data.Implementations;
+
+public class RoleDataService : DataService<Role>, IRoleDataService
+{
+    public RoleDataService(OjsDbContext db)
+        : base(db)
+    {
+    }
+}
```

Comment:
Do you need this service at all? Can you use `IDataService<Role>` directly where you need it?

Reply:
Fixed